src/wallet/connection.ts

```diff
@@ -10,13 +10,16 @@ import config from '../config';
 import BaseConnection, {
   DEFAULT_PARAMS,
   ConnectionParams,
+  IConnection,
 } from '../connection';
 import {
   WsTransaction,
   ConnectionState,
 } from './types';
+import { EventEmitter } from 'events';
 
 export interface WalletServiceConnectionParams extends ConnectionParams {
+  network?: string;
```

Comment:
Why did you add this here? `ConnectionParams` already has this parameter

Reply:
In the dummy, this was an optional param. That's why I added `?`, but sure, I will remove it.